web/tests/pm_list.test.cjs

```diff
@@ -31,3 +30,106 @@ run_test("update_dom_with_unread_counts", () => {
     assert.equal($total_count.text(), "");
     assert.equal($total_count.hasClass("hide"), true);
 });
+
+run_test("build_direct_messages_list", ({override}) => {
```

Comment:
I don't think we need this test addition. It's basically confirming that the boolean value for `all_conversations_shown` determines whether the `more_private_conversations_li` is rendered.